src/services/codefixes/importFixes.ts

```diff
@@ -1118,7 +1121,7 @@ function getUmdSymbol(token: Node, checker: TypeChecker): Symbol | undefined {
  * @internal
  */
 export function getImportKind(importingFile: SourceFile, exportKind: ExportKind, compilerOptions: CompilerOptions, forceImportKeyword?: boolean): ImportKind {
-    if (compilerOptions.verbatimModuleSyntax && (getEmitModuleKind(compilerOptions) === ModuleKind.CommonJS || importingFile.impliedNodeFormat === ModuleKind.CommonJS)) {
+    if (compilerOptions.verbatimModuleSyntax && getEmitModuleFormatOfFile(importingFile, compilerOptions) === ModuleKind.CommonJS) {

```

Comment:
All usage of options in this file as well need to be resolution options for file?

Reply:
I think you’re right, though not related to this PR; this is pretty much a direct translation

Reply:
I think I’d like to keep this for now—essentially none of the language service (and very little of the checker) ever looks up project reference options. It makes sense to me that we possibly should, but I think it would be a massive refactor and not related to this PR at all.

Reply:
I'm almost through making these changes, but it did occur to me while going through this file that `importingFile` should always belong to the program/options being used... if it belonged to a referenced project, a different LanguageService would have been called.